src/transport/AdminPairingTable.h

```diff
@@ -105,6 +110,7 @@ class DLL_EXPORT AdminPairingInfo
     NodeId mNodeId     = kUndefinedNodeId;
     FabricId mFabricId = kUndefinedFabricId;
     uint16_t mVendorId = kUndefinedVendorId;
+    char mFabricLabel[32];
```

Comment:
- Move 32 to a constant
- Make sure to leave +1 to make sure that a 32 octet label (since messages do NOT have null termination on strings, according to spec) has space for the null-terminator since string assumptions are made